### PR TITLE
bug 1682997 machine fixes

### PR DESCRIPTION
**Pull request by kalexand-rh**
From https://bugzilla.redhat.com/show_bug.cgi?id=1682997

@jhou1, PTAL?

### CODE REVIEW

**Comment by jhou1** [comment]
Looks good!

File: modules/machine-api-overview.adoc [hunk 1]
```diff
@@ -20,9 +20,6 @@ The three primary resources are:
 class, which describes the types of compute nodes that are offered for different
```
Comment (sunzhaohua2) [inline modules/machine-api-overview.adoc]: Line 17, The three primary resources are -> The two primary resources are

File: modules/machine-api-overview.adoc [hunk 2]
```diff
@@ -36,7 +33,7 @@ specified `MachineSet`, and the `MachineAutoscaler` maintains that range of node
 The `MachineAutoscaler` object takes effect after a `ClusterAutoscaler` object
 exists. Both `ClusterAutoscaler` and `MachineAutoscaler` resources are made
 available by the `ClusterAutoscalerOperator`.
-`MachineHealthChecker`:: This resource detects when a machine is unhealthy,
+`MachineHealthCheck`:: This resource detects when a machine is unhealthy,
 deletes it, and, on supported platforms, makes a new machine.
 `ClusterAutoscaler`:: This resource is based on the upstream
 link:https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler[ClusterAutoscaler]
```
Comment (sunzhaohua2) [inline modules/machine-api-overview.adoc]: Line 42, ClusterAutoscalerOperator is not a custom resource.

**Comment by kalexand-rh** [comment]
@sunzhaohua2, I made changes based on your comments. Will you PTAL?

**Comment by sunzhaohua2** [comment]
Looks good! Thanks.

**Pull request merged**